packages/iTwinUI-react/src/core/Dialog/DialogTitleBar.tsx

```diff
@@ -9,48 +9,78 @@ import { useTheme } from '@storybook/theming';
 import { IconButton } from '../Buttons';
 import '@itwin/itwinui-css/css/dialog.css';
 import { DialogContextProps, useDialogContext } from './DialogContext';
+import { DialogTitleBarTitle } from './DialogTitleBarTitle';
 
 export type DialogTitleBarProps = {
   /**
-   * Dialog title bar content.
+   * Dialog title bar content. If passed, then `title` prop is ignored.
    */
-  children: React.ReactNode;
+  children?: React.ReactNode;
+  /**
+   * Dialog title.
+   */
+  title?: React.ReactNode;
```

Comment:
Should we call this `titleText` or something? To avoid confusion with the native `title` attribute.

Reply:
Renamed